b2share/modules/records/serializers/response.py

```diff
@@ -64,25 +75,49 @@ def view(pid, record, code=200, headers=None, links_factory=None):
         return response
     return view
 
+def search_responsify(serializer, mimetype):
+    """Create a Records-REST search result response serializer.
+    :param serializer: Serializer instance.
+    :param mimetype: MIME type of response.
+    :returns: Function that generates a record HTTP response.
+    """
+    def view(pid_fetcher, search_result, code=200, headers=None, links=None,
+             item_links_factory=None):
+        response = current_app.response_class(
+            serializer.serialize_search(pid_fetcher, search_result,
+                                        links=links,
+                                        item_links_factory=item_links_factory),
+            mimetype=mimetype)
+        response.status_code = code
+        if headers is not None:
+            response.headers.extend(headers)
+
+        if links is not None:
+            add_link_header(response, links)
+
+        return response
+
+    return view
+
```

Comment:
Why was this method moved to b2share codebase since it is originally provided by invenio_search